RecoTracker/IterativeTracking/python/InitialStepPreSplitting_cff.py

```diff
@@ -125,6 +125,37 @@
 )
 initialStepTrackCandidatesPreSplitting.MeasurementTrackerEvent = 'MeasurementTrackerEventPreSplitting'
 
+from Configuration.ProcessModifiers.trackingMkFitInitialStepPreSplitting_cff import trackingMkFitInitialStepPreSplitting
+from RecoTracker.MkFit.mkFitGeometryESProducer_cfi import mkFitGeometryESProducer
+import RecoTracker.MkFit.mkFitHitConverter_cfi as mkFitHitConverter_cfi
+import RecoTracker.MkFit.mkFitSeedConverter_cfi as mkFitSeedConverter_cfi
+import RecoTracker.MkFit.mkFitIterationConfigESProducer_cfi as mkFitIterationConfigESProducer_cfi
+import RecoTracker.MkFit.mkFitProducer_cfi as mkFitProducer_cfi
+import RecoTracker.MkFit.mkFitOutputConverter_cfi as mkFitOutputConverter_cfi
+# TODO: Split the MkHitConverter to two to avoid re-converting strip hits in initialStep
+mkFitHitsPreSplitting = mkFitHitConverter_cfi.mkFitHitConverter.clone(
+    pixelRecHits = 'siPixelRecHitsPreSplitting'
```

Comment:
ah, this is one of the non-trivial parts of the pre-splitting setup; strips should be separated.

Reply:
something to discuss with DPG/POG is if the hits should eventually be moved to RecoLocalTracker/Configuration/

Reply:
Right. I'd anyway leave it for a subsequent PR to allow physics testing (and anyway the hit conversion time should be small compared to the improvement in pattern recognition time).